Assets/MixedRealityToolkit/_Core/Definitions/Utilities/MixedRealityPose.cs

```diff
@@ -47,7 +47,7 @@ public SixDof(Quaternion rotation)
         /// <returns>
         /// <see cref="Vector3.zero"/> and <see cref="Quaternion.identity"/>.
         /// </returns>
-        public static SixDof ZeroIdentity { get; } = new SixDof(Vector3.zero, Quaternion.identity);
+        public static MixedRealityPose ZeroIdentity { get; } = new MixedRealityPose(Vector3.zero, Quaternion.identity);
```

Comment:
Not fond of the name "ZeroIdentity" can we call it "Neutral" or something similar?

Reply:
Unity calls theirs `Pose.identity` Why their public properly is pascalCase I'll never know,

Reply:
Let's go with "Identity" as that is the common standard.  Although I personally liked ZeroIdentity as it did exactly what it said on the tin :D (A Zero Vector 3 with an Identity Quaternion)